src/Schedulers/SchedulerTraits.php

```diff
@@ -234,7 +234,11 @@ public static function get_next_blocking_job( $action_name ) {
 		if ( is_array( $blocking_jobs ) ) {
 			foreach ( $blocking_jobs as $blocking_job ) {
 				$blocking_job_hook = $blocking_job->get_hook();
-				$next_job_schedule = $blocking_job->get_schedule()->next();
+				if ( method_exists( $blocking_job->get_schedule(), 'get_date' ) ) {
```

Comment:
Curious why we use `get_date()` here- is `next()` not available to in progress actions?  And is `get_date()` not available to pending actions?

Reply:
`next()` logs a deprecated warning in 3.0. `get_date()` was added in 3.0.

Reply:
That makes sense - thanks for the explanation.